pkg/bindinfo/utils.go

```diff
@@ -167,6 +170,110 @@ func readBindingsFromStorage(sPool util.DestroyableSessionPool, condition string
 	return
 }
 
+var (
+	// UpdateBindingUsageInfoBatchSize indicates the batch size when updating binding usage info to storage.
+	UpdateBindingUsageInfoBatchSize = 20
```

Comment:
Updating 20 bindings (rows) each time seems too small from my experience, could we increase this number to like 100 or 200 since they are just simple update statements? Please help me confirm @you06